UC-Berkeley-Extension/global
# Create Voices Spotlight component

*This Pull request was built off of #27's `angular/info-sessions` branch. Do not merge before that one!*

## Asana Ticket(s)
- See ticket [Angular2 Templates Voices Posts](https://app.asana.com/0/1174970560648905/1197472455447399).

## Description
• Created Voices Spotlight Angular template
• Created [Voices Spotlight on the Styleguide](http://localhost:3002/?p=molecules-voices-spotlight)

## Testing instructions
* Checkout this branch
* Run `../scripts/setup.sh` to re-install Drupal
* Go to https://global.ddev.site/ 
 * Observe the `Angular: Voices Spotlight ` block is displayed on the homepage and has content.
* Login using `drush uli`
* Go to [Blocks](https://global.ddev.site/admin/structure/block)
* Observe that the `Angular: Voices Spotlight` block is enabled in the Content section.
* [Edit the block](https://global.ddev.site/admin/structure/block/manage/angularvoicespostspotlight?destination=/admin/structure/block)
* Observe that you can edit the following values:
  - post
  - areas
  - programs
* Specify you'd like to see post 1935.
* Go to the homepage
* Observe that the block only shows the COVID-19 Resource Guide for Students 

## Remaining Tasks
- [ ] Work on fallback when no posts exist
